src/test/test.cpp

```diff
@@ -199,8 +199,8 @@ getTestConfig(int instanceNumber, Config::TestDbMode mode)
         // faster. You can change this by enabling the appropriate line below
         // mode = Config::TESTDB_IN_MEMORY;
         // mode = Config::TESTDB_BUCKET_DB_PERSISTENT;
-        // mode = Config::TESTDB_POSTGRESQL;
-        mode = Config::TESTDB_BUCKET_DB_VOLATILE;
+        mode = Config::TESTDB_POSTGRESQL;
```

Comment:
I'm not sure I understand this change. Wouldn't it break the `USE_POSTGRES` macro semantics?